clients/mobile/Makefile

```diff
@@ -0,0 +1,81 @@
+# Define environment variables
+ENV_FILE=.env
+APP_NAME=net.bloodconnect.app
+EAS_CONFIG=eas.json
+
+# Default build profile (can be overridden via CLI)
+BUILD_PROFILE ?= development
+
+# Define default targets
+.PHONY: help install-deps login setup-env configure build clean
```

Comment:
how would we use this make file from the root makefile? I.e for example using it from the dev container?

Reply:
I think we can call like below
```
make -C ./clients/mobile upload-env BUILD_PROFILE=$(BUILD_PROFILE)
```

Reply:
i meant if we should have a target in the main makefile that can call this one?

Reply:
For now I ignored it bhai, I have to work on Melon bhai's branch as he done some work related to this...